src/components/tabController/TabBar.js

```diff
@@ -404,7 +408,7 @@ class TabBar extends PureComponent {
           snapToOffsets={this.getSnapBreakpoints()}
           decelerationRate={'fast'}
         >
-          <View style={[styles.tabBar, height && {height}, {paddingHorizontal: this.centerOffset}]}>
```

Comment:
Not sure why I applied the white background color both on the ScrollView and the inner View. I guess it's I forgot to remove.
I think it's redundant and we can keep it only on the ScrollView or the inner View.

Reply:
Actually it seems to work the same even when I remove the white background from them both..

Reply:
But then it will transparent, because the background color is undefined. 
What do you mean exactly?